src/common/transformations/src/transformations/fp16_compression/mark_decompression_convert_constant_folding.cpp

```diff
@@ -68,3 +68,28 @@ pass::KeepConstAndDecompression::KeepConstAndDecompression() {
     auto m = std::make_shared<pattern::Matcher>(node_pattern, matcher_name);
     register_matcher(m, callback);
 }
+
+pass::KeepConstAndDecompressionForMatMul::KeepConstAndDecompressionForMatMul() {
+    MATCHER_SCOPE(KeepConstAndDecompressionForMatMul);
+    auto matmul = pass::pattern::wrap_type<opset8::MatMul>();
```

Comment:
minor: it's better to use the latest available opset version, opset12 as I know

Reply:
updated, thanks